clients/fides-js/src/lib/i18n/i18n-utils.ts

```diff
@@ -265,6 +265,16 @@ export function matchAvailableLocales(
   return DEFAULT_LOCALE;
 }
 
+/**
+ * Check if the given message exists in the current locale's catalog.
+ */
+export function messageExists(i18n: I18n, id: string): boolean {
+  // NOTE: Our i18n library mirrors LinguiJS's implementation, which doesn't
+  // expose a way to check for missing messages directly. However, it will
+  // always return the "id" if no matching message exists, so we rely on that!
+  return i18n.t(id) !== "" && i18n.t(id) !== id;
```

Comment:
The `id` we pass into this method is the locale id?

Reply:
It's the message id - so things like `exp.title`, or `static.gpc.status.applied`